include/functions.h

```diff
@@ -1297,22 +1297,22 @@ void func_800A663C(GlobalContext* globalCtx, PSkinAwb*, SkeletonHeader*, Animati
 void func_800A6888(GlobalContext*, PSkinAwb*);
 // ? func_800A698C(?);
 // ? func_800A6AC4(?);
-// ? func_800A6E10(?);
-void func_800A6EF4(MtxF* mf, Vec3f* src, Vec3f* dest);
-void func_800A6FA0(MtxF* mf1, MtxF* mf2, MtxF* dest);
-// ? func_800A72FC(?);
-// ? func_800A730C(?);
-// ? func_800A735C(?);
+void SkinMatrix_Vec3fMtxFMultXYZW(MtxF* mf, Vec3f* src, Vec3f* xyzDest, f32* wDest);
+void SkinMatrix_Vec3fMtxFMultXYZ(MtxF* mf, Vec3f* src, Vec3f* dest);
+void SkinMatrix_MtxFMtxFMult(MtxF* mfB, MtxF* mfA, MtxF* dest);
+void SkinMatrix_GlobalIdentity(MtxF** mf);
+void SkinMatrix_SetToIdentity(MtxF* mf);
+void SkinMatrix_Copy(MtxF* src, MtxF* dest);
 // ? func_800A73E0(?);
-void func_800A76A4(MtxF* dest, f32 x, f32 y, f32 z);
-void func_800A7704(MtxF* dest, s16 x, s16 y, s16 z);
-// ? func_800A7894(?);
-void func_800A7A24(MtxF* dest, f32 x, f32 y, f32 z);
-// ? func_800A7B04(?);
-// ? func_800A7B84(?);
-// ? func_800A7C60(?);
+void SkinMatrix_Scale(MtxF* mf, f32 xScale, f32 yScale, f32 zScale);
+void SkinMatrix_RotateRPY(MtxF* mf, s16 roll, s16 pitch, s16 yaw);
+void SkinMatrix_RotateYRP(MtxF* mf, s16 yaw, s16 roll, s16 pitch);
```

Comment:
I could be wrong but some functions in here seem to be unused outside of the file?
If they aren't used globally they should be omitted from `functions.h`, just like `variables.h` is only meant for variables used globally.